test/verify/check-dashboard

```diff
@@ -316,7 +304,7 @@ class TestDashboardEditLimits(MachineCase, DashBoardHelpers):
 @nondestructive
 class TestLocalDashboard(MachineCase, DashBoardHelpers):
     def testDashboard(self):
-        self.restore_file("/etc/ssh/ssh_known_hosts")
+        self.restore_file("/home/admin/.ssh/known_hosts")
```

Comment:
Do we still have a place where we test the global known_hosts file?

Reply:
Good point.  Not very explicitly, I think.  I add one.

Reply:
Done.